dragscroll.js

```diff
@@ -1,11 +1,16 @@
 /**
- * @fileoverview dragscroll - scroll area by dragging
- * @version 0.0.8
- * 
- * @license MIT, see http://github.com/asvd/dragscroll
- * @copyright 2015 asvd <heliosframework@gmail.com> 
+ * @fileoverview dragscroll-zoom - scroll area by dragging and zoom in~out
+ * @version 1.0.0
+ *
+ * @license MIT, see https://github.com/cymakr/dragscroll-zoom
+ * @copyright 2017 cymakr <cymakr@gmail.com>
+ *
+ * This project began as a fork of dragscroll https://github.com/asvd/dragscroll
  */
-
+var zoom = 100;
```

Comment:
You have a lot of globals here, put everything into the factory method and export whatever needed.

(Someone else can change the `zoom` variable and your code breaks)